packages/editor-ui/src/modules/settings.ts

```diff
@@ -69,6 +78,35 @@ const module: Module<ISettingsState, IRootState> = {
 
 			context.commit('setPersonalizationAnswers', results);
 		},
+		async fetchPromptsData(context: ActionContext<ISettingsState, IRootState>) {
+			try {
+				const promptsData: IN8nPrompts = await getPromptsData(context.state.settings.instanceId);
```

Comment:
based on our meeting today, I understood that we won't show these modals if telemetry is not unabled. then we should avoid making this call completely if telemetry is disabled..

Reply:
Added.